portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+
+/** Class representing a comment. */
+public class Comment {
+  private final String author;
+  private final String commentContent;
+  private final Date date;
+
+  public Comment(String author, String commentContent, Date date) {
+    this.author = author;
+    this.commentContent = commentContent;
+    this.date = date;
+  }
+}
```

Comment:
How are you going to access the fields for server side inspection? I would recommend making this a typical Java Pojo https://en.wikipedia.org/wiki/Plain_old_Java_object
I.e. add getters for the fields (IntelliJ can autogenerate getters and setters for you). If it is a readonly class for now, you can skip the setters until needed.